greenpeace/planet4-docker
# Redirect sitemap.xml to wp-sitemap.xml

This is the actual url of the core sitemap feature.
But we need a redirection since many NROs have submitted `sitemap.xml` to Google Search Console.

This has to be merged after https://github.com/greenpeace/planet4-base/pull/182

### Testing

1. On a new docker-compose environment, you adjust [this line](https://github.com/greenpeace/planet4-docker-compose/blob/master/docker-compose.yml#L85) to `image: gcr.io/planet-4-151612/openresty:sitemap` (or just an environment variable.
2. Run `make dev`
3. Login and make sure that sitemap plugin is deactivated
4. Make sure you have unchecked "Discourage search engines from indexing this site" (otherwise no sitemap is being generated)
5. `/sitemap.xml` should redirect to `wp-sitemap.xml`
